external/PATCH-01-STRATUM

```diff
@@ -57,7 +57,7 @@ index e1f4338e..bffdc71e 100644
 +            "p4_programs": [
 +                {
 +                    "program-name": "dummy",
-+		    "cpu_numa_node": "0",
++                    "cpu_numa_node": "0",
```

Comment:
Could all the formatting changes not related to implementing TAP support be included in another PR? It is very difficult to see which code changed and to get to understand relations between code when there is so much clutter....

Reply:
@stolsma thanks for the review!! Since it was a minor change, I added it here. But you are right, I can create a separate PR for such changes